packages/connector-wechat-native/README.md

```diff
@@ -79,7 +106,9 @@ After passing the review, go to the application details page and generate an App
 }
 ```
 
-## Add the Logto social plugin to your app
+## Set up WeChat native sign in with your app
```

Comment:
I think we can just keep the original title since the whole article is talking about "Set up WeChat native sign in with your app", right?

Reply:
But this section contains iOS and Android, in Android, we don't use the `plugin`.
Change the title to `Add the Logto social support to your app`